src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -214,9 +215,11 @@ fi
 # Generate kernel u-boot image
 #
 IMAGE="$IMAGES_DIR/Image"
-IMAGE_UB="$BUILD_DIR/Image.ub"
 IMAGE_ELF_START="0x80000"
-$MKIMAGE -n 'Kernel Image' -A arm64 -O linux -C none -T kernel -C gzip -a $IMAGE_ELF_START -e $IMAGE_ELF_START -d $IMAGE $IMAGE_UB
+
+cp $IMAGE $BUILD_DIR/Image
+yes| gzip $BUILD_DIR/Image
+$MKIMAGE -n 'Kernel Image' -A arm64 -O linux -C none -T kernel -C gzip -a $IMAGE_ELF_START -e $IMAGE_ELF_START -d $BUILD_DIR/Image.gz $IMAGE_UB
```

Comment:
Why do we need to create the Kernel Image? why cant we generate this from Petalinux directly?

Reply:
Because system.dtb is from shell, so we have to use bootm <kernel> <rootfs> <system.dtb> to boot. Currently, petalinux does not generate u-boot format kernel but a raw Image. We can modify the script if petalinux provides u-boot format kernel later.